app/main/spawn.js

```diff
@@ -14,11 +14,15 @@ import { spawn } from 'child_process';
 import path from 'path';
 
 const UA_SERVICE_PATH = path.join(__dirname, '..', 'services', 'user-agent-service', 'index');
+const CONTENT_SERVICE_PATH = path.join(__dirname, '..', 'services', 'content-service', 'index');
 
 // @TODO Use something like `forever` or `pm2` to ensure that this
 // process remains running
 export function userAgentService() {
   spawn('node', [UA_SERVICE_PATH], {
     detached: true,
   });
+  spawn('node', [CONTENT_SERVICE_PATH], {
```

Comment:
Probably should be another exposed function so its clear in app/main/browser.js what services are being start up (both UA and Content)